dae-alright/src/container/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+
+
+class App extends Component {
+
+  render(){
+  return (
+    <div className="App">
+     
+    
```

Comment:
You don't seem to be rendering anything here, how did you test your nav bar?

Reply:
I actually removed it from the App.js before pushing otherwise it'll throw conflicts